snf-cyclades-app/synnefo/api/util.py

```diff
@@ -188,6 +189,13 @@ def get_image(image_id, user_id):
             raise faults.ItemNotFound("Image '%s' not found" % image_id)
 
 
+def get_keypair(keypair_name, user_id):
```

Comment:
This should support argument `for_update=False` (see other get functions in the same file). When `True`, it should do the query with `select_for_update()` which is needed for any API view that updates the retrieved object.